### ISSUE TITLE
Allow user defined icons with sky-label

### ISSUE THREAD

**Issue by Blackbaud-BrianBlanchard** [blackbaud/skyux-indicators#59]
### Expected behavior
Allow icons to be defined by the user.  We'd like to use an info label, but have the icon be a paperclip instead:

![image](https://user-images.githubusercontent.com/23219799/74748863-3428f600-5237-11ea-8ff2-5cde056be653.png)

### Actual behavior
Cannot use sky-label and need to use custom css to display as intended.

Feel free to close, but it would be nice if this behavior came with the sky-label component.

### Severity
Low

### Impact
Low

**Comment by Blackbaud-ToddRoberts** [comment]
What is the context of use? Currently labels are intended to only be used with the provided icons.

**Comment by Blackbaud-BrianBlanchard** [comment]
See the screenshot.  Our designer mocked that up, and `sky-label` seemed like the best fit for that design.  Without this support, we'll be forced to re-invent the `sky-label` styling ourselves.

**Comment by Blackbaud-ToddRoberts** [comment]
do you have a full page screenshot of where it's being used?

**Comment by Blackbaud-BrianBlanchard** [comment]
no, its in development - but here's the mockup we're using: https://blackbaud.invisionapp.com/share/HJUXK7P4WNC#/screens/400139686_actions_Tile

**Comment by Blackbaud-ToddRoberts** [comment]
This would need a design proposal since the usage does not follow current guidelines for labels. You can have the designer get in contact with me.

**Comment by Blackbaud-BrianBlanchard** [comment]
So your recommendation is to re-implement the existing sky-label component with our own icons?

**Comment by Blackbaud-DanMiller** [comment]
waiting on proposal from UX

**Comment by Blackbaud-SteveBrush** [comment]
Closing issue due to inactivity. If you believe this should still be addressed, please file a new issue and include any additional context and justification. Thank you.
